src/clj/clojure/lang/persistent_sorted_set.clj

```diff
@@ -1,18 +1,11 @@
 (ns clojure.lang.persistent-sorted-set
-  (:refer-clojure :only [apply cons declare defn defn- sorted-map sorted-map-by])
-  (:require [clojure.lang.apersistent-set :refer [defset make-pairs]]))
+  (:refer-clojure :only [apply cons declare defn defn-])
+  (:require [clojure.lang.apersistent-set :refer [defset]]))
 
-(declare new-sorted-set)
+(declare make-sorted-set)
 
-(defset PersistentSortedSet new-sorted-set)
+(defset PersistentSortedSet make-sorted-set)
 
-(defn new-sorted-set [sorted-map]
+(defn make-sorted-set [sorted-map]
```

Comment:
It looks like you're differentiating between `new` and `make`, what is the rule you're following?